ckanext/dcatapchharvest/profiles.py

```diff
@@ -298,9 +328,10 @@ def parse_dataset(self, dataset_dict, dataset_ref):  # noqa
         )
 
         # Publisher
-        dataset_dict['publishers'] = self._publishers(
+        dataset_dict['publisher'] = self._publisher(
             dataset_ref,
-            DCT.publisher
+            DCT.publisher,
+            dataset_dict['identifier']
```

Comment:
Do we really need to use 'identifier' here, shouldn't it be `organization_url` ?

Reply:
in the migration you use as an url for publisher: ORGANIZATION_URI_BASE + organization_name

Reply:
what happened if an  "identifier" = ""

Reply:
Yes that is true, but I noticed that at the time we parse the datasets we don't have the organisation content. It is always just derived from the identifier and added later in the harvesting process not during the parsing, see here: https://github.com/opendata-swiss/ckanext-dcatapchharvest/blob/cee6654127ca5b28dc301783ccad71e8b12781e7/ckanext/dcatapchharvest/harvesters.py#L83